labs/projecteuler/projecteuler.md

```diff
@@ -0,0 +1,132 @@
+>    * d_(8)d_(9)d_(10)=289 is divisible by 17
+>
+> Find at least two 0 to 9 pandigital numbers with this property.  
+
+### [Problem 48](https://projecteuler.net/problem=48)
+
+> The series, 1^(1) + 2^(2) + 3^(3) + ... + 10^(10) = 10405071317.
+>
+> Find the last ten digits of the series, 1^(1) + 2^(2) + 3^(3) + ... + 1000^(1000).
+
+### [Problem 52](https://projecteuler.net/problem=52)
+
+>It can be seen that the number, 125874, and its double, 251748, contain exactly the same digits, but in a different order.
+>
+> Find the smallest positive integer, x, such that 2x, 3x, 4x, 5x, and 6x, contain the same digits. 
+
+### [Problem 59](https://projecteuler.net/problem=59)
+
+> Each character on a computer is assigned a unique code and the preferred standard is ASCII (American Standard Code for Information Interchange). For example, uppercase A = 65, asterisk (*) = 42, and lowercase k = 107.
+>
+> A modern encryption method is to take a text file, convert the bytes to ASCII, then XOR each byte with a given value, taken from a secret key. The advantage with the XOR function is that using the same encryption key on the cipher text, restores the plain text; for example, 65 XOR 42 = 107, then 107 XOR 42 = 65.
+>
+> For unbreakable encryption, the key is the same length as the plaintext message, and the key is made up of random bytes. The user would keep the encrypted message and the encryption key in different locations, and without both "halves", it is impossible to decrypt the message.
+>
+> Unfortunately, this method is impractical for most users, so the modified method is to use a password as a key. If the password is shorter than the message, which is likely, the key is repeated cyclically throughout the message. The balance for this method is using a sufficiently long password key for security, but short enough
+to be memorable.
+>
+> Your task has been made easy, as the encryption key consists of three lower case characters. Using cipher1.cry, a file containing the encrypted ASCII codes, and the knowledge that the plain text must contain common English words, decrypt the message and find the sum of the ASCII values in the original text.
+
+### [Problem 79](https://projecteuler.net/problem=79)
+
+> A common security method used for online banking is to ask the user for three random characters from a passcode. For example, if the passcode was 531278, they may asked for the 2nd, 3rd, and 5th characters; the expected reply would be: 317.
+>
+> The text file, keylog.cry, contains fifty successful login attempts.
+>
+> Given that the three characters are always asked for in order, analyse the file so as to determine the shortest possible secret passcode of >unknown length.
+
+### Project Sean
+Sean Weaver (Super Genius ala Wile E Coyote) has a problem in a similar vein:
+
+> Find a four-digit number (greater than 999 and less than 10000) such that the least significant four digits of the square is that number.
```

Comment:
"...four-digit **decimal** number..."  (for the pedants in the crowd)

Reply:
I'm keeping it as is.  I'm willing to be outvoted.